tests/unit/cache.test.ts

```diff
@@ -0,0 +1,24 @@
+import { assert } from "deno/testing/asserts.ts"
+import { useDownload } from "hooks"
+import { sandbox } from "../utils.ts"
+
+Deno.test("etag-mtime-check",async () => {
+  await sandbox(async tmpdir => {
+    const src = new URL("https://dist.tea.xyz/ijg.org/versions.txt")
+    await useDownload().download({src, dst: tmpdir.join("test.tar.xz")})
```

Comment:
the `dst` is wrong. Just remove it.

Reply:
if there is no `dst` specified it will download it to `tea` directory i.e. `~/.tea/tea.xyz/var/www/https/dist.tea.xyz`

Reply:
well I guess you're right in that you shouldn't do that, but you are currently downloading to a file called `test.tar.xz` which is flagrantly wrong. Anyone reading this code in the future will wonder what the fuck you were thinking.